2025-10-12 23:53:44 [error]: Database connection failed {"error":"connect ECONNREFUSED ::1:5432"}
2025-10-12 23:56:09 [error]: Database connection failed {"error":"connect ECONNREFUSED ::1:6543"}
2025-10-12 23:56:29 [error]: Database connection failed {"error":"connect ECONNREFUSED ::1:6543"}
2025-10-12 23:57:53 [error]: Database connection failed {"error":"connect ECONNREFUSED ::1:6543"}
2025-10-12 23:59:27 [error]: Database connection failed {"error":"connect ECONNREFUSED ::1:6543"}
2025-10-12 23:59:52 [error]: Database connection failed {"error":"connect ECONNREFUSED ::1:6543"}
2025-10-13 00:11:26 [error]: Database connection failed {"error":"connect ECONNREFUSED ::1:6543"}
2025-10-13 00:38:57 [error]: Post.fromDatabase is not a function {"path":"/api/v1/posts","method":"GET","ip":"127.0.0.1"}
TypeError: Post.fromDatabase is not a function
    at file:///C:/Users/HOME/Desktop/site/taiginsky/backend/src/infrastructure/repositories/PostRepository.js:29:37
    at Array.map (<anonymous>)
    at PostRepository.findAll (file:///C:/Users/HOME/Desktop/site/taiginsky/backend/src/infrastructure/repositories/PostRepository.js:29:18)
    at async GetPostsUseCase.execute (file:///C:/Users/HOME/Desktop/site/taiginsky/backend/src/application/use-cases/post/GetPostsUseCase.js:12:19)
    at async getPosts (file:///C:/Users/HOME/Desktop/site/taiginsky/backend/src/presentation/controllers/PostController.js:34:21)
2025-10-13 00:38:57 [error]: Post.fromDatabase is not a function {"path":"/api/v1/posts","method":"GET","ip":"127.0.0.1"}
TypeError: Post.fromDatabase is not a function
    at file:///C:/Users/HOME/Desktop/site/taiginsky/backend/src/infrastructure/repositories/PostRepository.js:29:37
    at Array.map (<anonymous>)
    at PostRepository.findAll (file:///C:/Users/HOME/Desktop/site/taiginsky/backend/src/infrastructure/repositories/PostRepository.js:29:18)
    at async GetPostsUseCase.execute (file:///C:/Users/HOME/Desktop/site/taiginsky/backend/src/application/use-cases/post/GetPostsUseCase.js:12:19)
    at async getPosts (file:///C:/Users/HOME/Desktop/site/taiginsky/backend/src/presentation/controllers/PostController.js:34:21)
2025-10-13 00:39:07 [error]: Post.fromDatabase is not a function {"path":"/api/v1/posts","method":"GET","ip":"127.0.0.1"}
TypeError: Post.fromDatabase is not a function
    at file:///C:/Users/HOME/Desktop/site/taiginsky/backend/src/infrastructure/repositories/PostRepository.js:29:37
    at Array.map (<anonymous>)
    at PostRepository.findAll (file:///C:/Users/HOME/Desktop/site/taiginsky/backend/src/infrastructure/repositories/PostRepository.js:29:18)
    at async GetPostsUseCase.execute (file:///C:/Users/HOME/Desktop/site/taiginsky/backend/src/application/use-cases/post/GetPostsUseCase.js:12:19)
    at async getPosts (file:///C:/Users/HOME/Desktop/site/taiginsky/backend/src/presentation/controllers/PostController.js:34:21)
2025-10-13 00:39:07 [error]: Post.fromDatabase is not a function {"path":"/api/v1/posts","method":"GET","ip":"127.0.0.1"}
TypeError: Post.fromDatabase is not a function
    at file:///C:/Users/HOME/Desktop/site/taiginsky/backend/src/infrastructure/repositories/PostRepository.js:29:37
    at Array.map (<anonymous>)
    at PostRepository.findAll (file:///C:/Users/HOME/Desktop/site/taiginsky/backend/src/infrastructure/repositories/PostRepository.js:29:18)
    at async GetPostsUseCase.execute (file:///C:/Users/HOME/Desktop/site/taiginsky/backend/src/application/use-cases/post/GetPostsUseCase.js:12:19)
    at async getPosts (file:///C:/Users/HOME/Desktop/site/taiginsky/backend/src/presentation/controllers/PostController.js:34:21)
2025-10-13 00:39:13 [error]: Post.fromDatabase is not a function {"path":"/api/v1/posts","method":"GET","ip":"127.0.0.1"}
TypeError: Post.fromDatabase is not a function
    at file:///C:/Users/HOME/Desktop/site/taiginsky/backend/src/infrastructure/repositories/PostRepository.js:29:37
    at Array.map (<anonymous>)
    at PostRepository.findAll (file:///C:/Users/HOME/Desktop/site/taiginsky/backend/src/infrastructure/repositories/PostRepository.js:29:18)
    at async GetPostsUseCase.execute (file:///C:/Users/HOME/Desktop/site/taiginsky/backend/src/application/use-cases/post/GetPostsUseCase.js:12:19)
    at async getPosts (file:///C:/Users/HOME/Desktop/site/taiginsky/backend/src/presentation/controllers/PostController.js:34:21)
2025-10-13 00:39:13 [error]: Post.fromDatabase is not a function {"path":"/api/v1/posts","method":"GET","ip":"127.0.0.1"}
TypeError: Post.fromDatabase is not a function
    at file:///C:/Users/HOME/Desktop/site/taiginsky/backend/src/infrastructure/repositories/PostRepository.js:29:37
    at Array.map (<anonymous>)
    at PostRepository.findAll (file:///C:/Users/HOME/Desktop/site/taiginsky/backend/src/infrastructure/repositories/PostRepository.js:29:18)
    at async GetPostsUseCase.execute (file:///C:/Users/HOME/Desktop/site/taiginsky/backend/src/application/use-cases/post/GetPostsUseCase.js:12:19)
    at async getPosts (file:///C:/Users/HOME/Desktop/site/taiginsky/backend/src/presentation/controllers/PostController.js:34:21)
2025-10-13 00:39:39 [error]: Post.fromDatabase is not a function {"path":"/api/v1/posts","method":"GET","ip":"127.0.0.1"}
TypeError: Post.fromDatabase is not a function
    at file:///C:/Users/HOME/Desktop/site/taiginsky/backend/src/infrastructure/repositories/PostRepository.js:29:37
    at Array.map (<anonymous>)
    at PostRepository.findAll (file:///C:/Users/HOME/Desktop/site/taiginsky/backend/src/infrastructure/repositories/PostRepository.js:29:18)
    at async GetPostsUseCase.execute (file:///C:/Users/HOME/Desktop/site/taiginsky/backend/src/application/use-cases/post/GetPostsUseCase.js:12:19)
    at async getPosts (file:///C:/Users/HOME/Desktop/site/taiginsky/backend/src/presentation/controllers/PostController.js:34:21)
2025-10-13 00:39:39 [error]: Post.fromDatabase is not a function {"path":"/api/v1/posts","method":"GET","ip":"127.0.0.1"}
TypeError: Post.fromDatabase is not a function
    at file:///C:/Users/HOME/Desktop/site/taiginsky/backend/src/infrastructure/repositories/PostRepository.js:29:37
    at Array.map (<anonymous>)
    at PostRepository.findAll (file:///C:/Users/HOME/Desktop/site/taiginsky/backend/src/infrastructure/repositories/PostRepository.js:29:18)
    at async GetPostsUseCase.execute (file:///C:/Users/HOME/Desktop/site/taiginsky/backend/src/application/use-cases/post/GetPostsUseCase.js:12:19)
    at async getPosts (file:///C:/Users/HOME/Desktop/site/taiginsky/backend/src/presentation/controllers/PostController.js:34:21)
2025-10-13 01:02:39 [error]: Do not know how to serialize a BigInt {"path":"/api/v1/ads","method":"GET","ip":"127.0.0.1"}
TypeError: Do not know how to serialize a BigInt
    at JSON.stringify (<anonymous>)
    at stringify (C:\Users\HOME\Desktop\site\taiginsky\backend\node_modules\express\lib\response.js:1160:12)
    at ServerResponse.json (C:\Users\HOME\Desktop\site\taiginsky\backend\node_modules\express\lib\response.js:271:14)
    at file:///C:/Users/HOME/Desktop/site/taiginsky/backend/src/presentation/controllers/AdController.js:34:32
2025-10-13 01:02:42 [error]: Do not know how to serialize a BigInt {"path":"/api/v1/ads","method":"GET","ip":"127.0.0.1"}
TypeError: Do not know how to serialize a BigInt
    at JSON.stringify (<anonymous>)
    at stringify (C:\Users\HOME\Desktop\site\taiginsky\backend\node_modules\express\lib\response.js:1160:12)
    at ServerResponse.json (C:\Users\HOME\Desktop\site\taiginsky\backend\node_modules\express\lib\response.js:271:14)
    at file:///C:/Users/HOME/Desktop/site/taiginsky/backend/src/presentation/controllers/AdController.js:34:32
2025-10-13 01:02:44 [error]: Do not know how to serialize a BigInt {"path":"/api/v1/ads","method":"GET","ip":"127.0.0.1"}
TypeError: Do not know how to serialize a BigInt
    at JSON.stringify (<anonymous>)
    at stringify (C:\Users\HOME\Desktop\site\taiginsky\backend\node_modules\express\lib\response.js:1160:12)
    at ServerResponse.json (C:\Users\HOME\Desktop\site\taiginsky\backend\node_modules\express\lib\response.js:271:14)
    at file:///C:/Users/HOME/Desktop/site/taiginsky/backend/src/presentation/controllers/AdController.js:34:32
2025-10-13 01:04:00 [error]: Uncaught Exception {"error":"listen EADDRINUSE: address already in use 0.0.0.0:4000"}
Error: listen EADDRINUSE: address already in use 0.0.0.0:4000
    at Server.setupListenHandle [as _listen2] (node:net:1751:16)
    at listenInCluster (node:net:1799:12)
    at doListen (node:net:1948:7)
    at process.processTicksAndRejections (node:internal/process/task_queues:83:21)
2025-10-13 09:24:29 [error]: Uncaught Exception {"error":"listen EADDRINUSE: address already in use 0.0.0.0:4000"}
Error: listen EADDRINUSE: address already in use 0.0.0.0:4000
    at Server.setupListenHandle [as _listen2] (node:net:1751:16)
    at listenInCluster (node:net:1799:12)
    at doListen (node:net:1948:7)
    at process.processTicksAndRejections (node:internal/process/task_queues:83:21)
2025-10-13 09:24:34 [error]: Uncaught Exception {"error":"listen EADDRINUSE: address already in use 0.0.0.0:4000"}
Error: listen EADDRINUSE: address already in use 0.0.0.0:4000
    at Server.setupListenHandle [as _listen2] (node:net:1751:16)
    at listenInCluster (node:net:1799:12)
    at doListen (node:net:1948:7)
    at process.processTicksAndRejections (node:internal/process/task_queues:83:21)
2025-10-13 09:25:01 [error]: Uncaught Exception {"error":"listen EADDRINUSE: address already in use 0.0.0.0:4000"}
Error: listen EADDRINUSE: address already in use 0.0.0.0:4000
    at Server.setupListenHandle [as _listen2] (node:net:1751:16)
    at listenInCluster (node:net:1799:12)
    at doListen (node:net:1948:7)
    at process.processTicksAndRejections (node:internal/process/task_queues:83:21)
2025-10-13 09:25:37 [error]: Uncaught Exception {"error":"listen EADDRINUSE: address already in use 0.0.0.0:4000"}
Error: listen EADDRINUSE: address already in use 0.0.0.0:4000
    at Server.setupListenHandle [as _listen2] (node:net:1751:16)
    at listenInCluster (node:net:1799:12)
    at doListen (node:net:1948:7)
    at process.processTicksAndRejections (node:internal/process/task_queues:83:21)
2025-10-13 09:26:52 [error]: Uncaught Exception {"error":"listen EADDRINUSE: address already in use 0.0.0.0:4000"}
Error: listen EADDRINUSE: address already in use 0.0.0.0:4000
    at Server.setupListenHandle [as _listen2] (node:net:1751:16)
    at listenInCluster (node:net:1799:12)
    at doListen (node:net:1948:7)
    at process.processTicksAndRejections (node:internal/process/task_queues:83:21)
2025-10-13 09:31:56 [error]: Uncaught Exception {"error":"listen EADDRINUSE: address already in use 0.0.0.0:4000"}
Error: listen EADDRINUSE: address already in use 0.0.0.0:4000
    at Server.setupListenHandle [as _listen2] (node:net:1751:16)
    at listenInCluster (node:net:1799:12)
    at doListen (node:net:1948:7)
    at process.processTicksAndRejections (node:internal/process/task_queues:83:21)
2025-10-13 09:32:07 [error]: Uncaught Exception {"error":"listen EADDRINUSE: address already in use 0.0.0.0:4000"}
Error: listen EADDRINUSE: address already in use 0.0.0.0:4000
    at Server.setupListenHandle [as _listen2] (node:net:1751:16)
    at listenInCluster (node:net:1799:12)
    at doListen (node:net:1948:7)
    at process.processTicksAndRejections (node:internal/process/task_queues:83:21)
2025-10-13 09:32:26 [error]: Uncaught Exception {"error":"listen EADDRINUSE: address already in use 0.0.0.0:4000"}
Error: listen EADDRINUSE: address already in use 0.0.0.0:4000
    at Server.setupListenHandle [as _listen2] (node:net:1751:16)
    at listenInCluster (node:net:1799:12)
    at doListen (node:net:1948:7)
    at process.processTicksAndRejections (node:internal/process/task_queues:83:21)
2025-10-13 09:32:28 [error]: Uncaught Exception {"error":"listen EADDRINUSE: address already in use 0.0.0.0:4000"}
Error: listen EADDRINUSE: address already in use 0.0.0.0:4000
    at Server.setupListenHandle [as _listen2] (node:net:1751:16)
    at listenInCluster (node:net:1799:12)
    at doListen (node:net:1948:7)
    at process.processTicksAndRejections (node:internal/process/task_queues:83:21)
2025-10-13 12:00:00 [error]: Cron job: Archive old ads failed {"error":"fetch failed"}
2025-10-13 12:22:27 [error]: Uncaught Exception {"error":"listen EADDRINUSE: address already in use 0.0.0.0:4000"}
Error: listen EADDRINUSE: address already in use 0.0.0.0:4000
    at Server.setupListenHandle [as _listen2] (node:net:1751:16)
    at listenInCluster (node:net:1799:12)
    at doListen (node:net:1948:7)
    at process.processTicksAndRejections (node:internal/process/task_queues:83:21)
